static/index.html

```diff
@@ -26,21 +25,23 @@
               <span class="hamburger" onclick="openNav()">&#9776;</span>
               <div id="side-navigation" class="menu-wrapper" >
                 <div class="close-button col-2 col-sm-3 col-md-2"><a href="javascript:void(0)" onclick="closeNav()">&times;</a></div>
-                  <ul class="menu-list col-12 col-md-12">
-                    <li class="menu-margin-top"><a href="#">venue</a></li>
-                    <li><a href="#">sponsors</a></li>
-                    <li><a href="#">pricing</a></li><br><br>
-                  </ul>
-                <div class="register-button col-12"><a href="#">register</a></div>
+                <ul class="list-group" col-12 col-md-12">
+                  <li class="list-group-item"><a href="#">Venue</a></li>
+                  <li class="list-group-item"><a href="#">Sponsors</a></li>
+                  <li class="list-group-item"><a href="#">Pricing</a></li>
+                </ul>
+                <button type="button" class="btn btn-primary">Register</button>
```

Comment:
This needs to be a link I think. Where would we place the link to /register?

Reply:
@alvincases , please wrap 'Register' in <a></a> tag.